app/scripts/src/content/search.js

```diff
@@ -59,20 +59,24 @@ Search.prototype = {
   findEpisode: function(options) {
     this.findItem({
       success: function(response) {
-        Request.send({
-          method: 'GET',
-          url: this.getEpisodeUrl(response['show']['ids']['slug']),
-          success: function(resp) {
-            if (this.item.episode) {
-              options.success.call(this, JSON.parse(resp));
-            } else {
-              this.findEpisodeByTitle(resp, options);
+        if (!response) {
```

Comment:
I see. But we don't know a case yet where the `response` will be `undefined`, right?
It looks very strange to me to call the error callback inside the success one, and I don't wanna add that logic unless it is needed.